### DISCUSSION TITLE
Detections: Sort suricata rules by category

### DISCUSSION THREAD

**Discussion by security-companion** [Security-Onion-Solutions/securityonion#14059] [Unsupported Versions] [answered]
Hi,
I noticed that some suricata rules have severity unknown.
From my understanding in this case the severity is derived from the classification of a specific rules as defined in https://github.com/Security-Onion-Solutions/securityonion/blob/ace6c5c9e4118653c83f414b4a7b0147672acd64/salt/suricata/classification/classification.config
So my question is if it is possible to sort and filter in detections by the classification/classtype (eg. all having value attempted-user) or even better by the value assigned to the classification in classification.config

Best regards
security-companion

**Comment by dougburks** [comment, marked as answer]
Have you tried the following query in Detections?
```
"classtype:attempted-user"
```

![image](https://github.com/user-attachments/assets/92618205-cf84-4771-9a4b-b7ccfe3d2f01)

**Comment by security-companion** [comment]
Hey,
thank you very much, that worked.
Greetings
